### PR TITLE
Added Jetstream profile translation for ja

### PR DESCRIPTION
**Pull request by jimbaran5601406**

### CODE REVIEW

**Comment by caouecs** [comment]
two translations for japanese. Can you @jimbaran5601406  and @yuusakuri work together ? Thank you

**Comment by jimbaran5601406** [comment]
No. I added some translations after looking at his translations. Therefore my commit may have a part of his translation.

**Comment by yuusakuri** [comment]
The translation of @ jimbaran5601406 has modified my translation and added more translations.
So adopt his translation.
My pull request closes.

**Comment by yuusakuri** [comment]
There was a change request in my pull request.
Would you please change these translations here?
[my pull request](https://github.com/Laravel-Lang/lang/pull/1460)

**Comment by jimbaran5601406** [comment]
@yuusakuri 
Thank you for teaching.
I just deleted it instead of adding the translation because it's unnecessary.

**Comment by yuusakuri** [comment]
@jimbaran5601406 
I'm sorry, my pull request has two commits since the first commit.
So please correct the translation by referring to the commit below.

[commit 2](https://github.com/Laravel-Lang/lang/pull/1460/commits/46f0bef6c3fd9b95dda19cfc8ba096ac9d4bceb6)
[commit 3](https://github.com/Laravel-Lang/lang/pull/1460/commits/05fd57023bd3ada345d7ba72f7a45a40efd173ef)

**Comment by caouecs** [comment]
> @yuusakuri
> Thank you for teaching.
> I just deleted it instead of adding the translation because it's unnecessary.

if you delete translations, it will be replaced by nothing on your website, so you can see doc for exclude translations : https://github.com/Laravel-Lang/lang/blob/master/CONTRIBUTING.md#what-should-i-do-if-there-is-a-tag-whose-translation-is-the-same-as-in-english

**Comment by jimbaran5601406** [comment]
@caouecs 
I'm sorry for misunderstanding.
I excluded the phrase "All rights reserved." and modified some translations by the yuusakuri's comment in the last commit.
Thank you for reviewing.

File: json/ja.json [hunk 1]
```diff
@@ -1,64 +1,125 @@
...
+    "Are you sure you want to delete your account? Once your account is deleted, all of its resources and data will be permanently deleted. Please enter your password to confirm you would like to permanently delete your account.": "本当にアカウントを削除してもよろしいでしょうか？ 一度アカウントを削除するとすべてのデータは永久に削除されます。よろしければ確認のためパスワードを入力してください。",
+    "Are you sure you would like to delete this API token?": "本当にAPIトークンを削除してよろしいでしょうか？",
     "Before proceeding, please check your email for a verification link.": "次に進む前に、確認リンクのメールをチェックしてください。",
+    "Browser Sessions": "ブラウザセッション",
     "click here to request another": "ここをクリックしてメールを再送信してください。",
+    "Close": "閉じる",
+    "Code": "コード",
+    "Confirm": "確認",
     "Confirm Password": "パスワード（確認用）",
+    "Create": "",
```
Comment (caouecs) [inline json/ja.json:19]: Need translation or in exclude file

File: json/ja.json [hunk 2]
```diff
@@ -1,64 +1,125 @@
...
+    "Create API Token": "APIトークン生成",
+    "Create New Team": "新しいチーム作成",
+    "Created.": "",
```
Comment (caouecs) [inline json/ja.json:22]: Need translation or in exclude file

File: json/ja.json [hunk 3]
```diff
@@ -1,64 +1,125 @@
...
     "If you did not receive the email": "メールが届かなかった場合",
     "If you did not request a password reset, no further action is required.": "パスワードのリセットにお心当たりが無い場合は、このメールを無視してください。",
     "If you’re having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:": "\":actionText\" ボタンが機能しない場合は, 次のURLをコピーしてWebブラウザに貼り付けてください。[:displayableActionUrl](:actionURL)",
+    "Last active": "最後のログイン",
     "Login": "ログイン",
     "Logout": "ログアウト",
+    "Logout Other Browser Sessions": "すべての端末からログアウト",
+    "Manage Account": "アカウント管理",
+    "Manage and logout your active sessions on other browsers and devices.": "すべての端末からログアウトする。",
+    "Manage API Tokens": "",
```
Comment (caouecs) [inline json/ja.json:54]: Need translation or in exclude file

File: json/ja.json [hunk 4]
```diff
@@ -1,64 +1,125 @@
...
     "Oh no": "オーノー",
+    "Once your account is deleted, all of its resources and data will be permanently deleted. Before deleting your account, please download any data or information that you wish to retain.": "一度あなたのアカウントが削除されると、すべてのデータはは永久に削除されます。削除する前に、保存しておきたいデータをダウンロードしてください。",
     "Page Expired": "期限切れのページ",
     "Page Not Found": "ページが見つかりません",
     "Password": "パスワード",
-    "Please click the button below to verify your email address.": "メールアドレスを確認するには、下のボタンをクリックしてください。",
-    "Please confirm access to your account by entering one of your emergency recovery codes.": "Please confirm access to your account by entering one of your emergency recovery codes.",
-    "Please confirm access to your account by entering the authentication code provided by your authenticator application.": "Please confirm access to your account by entering the authentication code provided by your authenticator application.",
+    "Permanently delete your account.": "永久にアカウントを削除する。",
+    "Permissions": "",
```
Comment (caouecs) [inline json/ja.json:65]: Need translation or in exclude file

File: json/ja.json [hunk 5]
```diff
@@ -1,64 +1,125 @@
...
+    "Profile": "プロフィール",
+    "Profile Information": "プロフィール情報",
+    "Recovery Code": "リカバリーコード",
     "Regards": "よろしくお願いします",
+    "Regenerate Recovery Codes": "リカバリーコード再生成",
     "Register": "アカウント作成",
-    "Remember Me": "ログイン状態を保持する",
-    "Resend Verification Email": "Resend Verification Email",
-    "Reset Password Notification": "パスワードリセット通知",
+    "Remember me": "ログイン状態を保持する",
```
Comment (caouecs) [inline json/ja.json:78]: We need "Remember Me" and "Remember me"

**Review by caouecs — CHANGES_REQUESTED** [review]
Thank you but some fixes to do

**Comment by jimbaran5601406** [comment]
@caouecs 
I fixed some and added more translations for ja.
Sorry for many times.

**Comment by caouecs** [comment]
Thank you

**Pull request merged**